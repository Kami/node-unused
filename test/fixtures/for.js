// variable declared in for look initializer
// goes unused in the body

for (var i=0, j=5 ; i<10 ; ++i) {
}
