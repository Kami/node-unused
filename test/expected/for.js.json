[
  {
    "name": "j",
    "loc": {
      "line": 4,
      "column": 14
    },
    "param": false
  }
]